packages/mobile/src/screens/sign-on-screen/components/SignUpFlowTwitterAuth.tsx

```diff
@@ -82,6 +82,9 @@ const useSetProfileFromTwitter = () => {
       twitterApi.headers,
       twitterApi.credentialsType
     )
+    if (tokenRes.error) {
+      throw new Error(tokenRes.error)
```

Comment:
I assume `tokenRes.error` is not already a throwable?

Reply:
yeah iirc error is just a string